From 1cd25c9a6574a4096c4e87b2c978c4cb08a92a31 Mon Sep 17 00:00:00 2001
From: Catalin Enache <catalin.enache@windriver.com>
Date: Fri, 20 Apr 2018 13:41:22 +0300
Subject: [PATCH 2/2] Revert "netatop: Add Kconfig and Makefile"

This reverts commit 027b6f65082dad1cd706393d8bd965c33d4d19d8.

This is removing an out of tree kernel module that isn't going
to be integrated into linux-yocto.

Issue: LIN10-3713
Signed-off-by: Catalin Enache <catalin.enache@windriver.com>
---
 drivers/staging/Kconfig          |  2 --
 drivers/staging/Makefile         |  1 -
 drivers/staging/netatop/Kconfig  | 13 -------------
 drivers/staging/netatop/Makefile |  1 -
 4 files changed, 17 deletions(-)
 delete mode 100644 drivers/staging/netatop/Kconfig
 delete mode 100644 drivers/staging/netatop/Makefile

diff --git a/drivers/staging/Kconfig b/drivers/staging/Kconfig
index ffe16fb..268d4e6 100644
--- a/drivers/staging/Kconfig
+++ b/drivers/staging/Kconfig
@@ -110,6 +110,4 @@ source "drivers/staging/ccree/Kconfig"
 
 source "drivers/staging/typec/Kconfig"
 
-source "drivers/staging/netatop/Kconfig"
-
 endif # STAGING
diff --git a/drivers/staging/Makefile b/drivers/staging/Makefile
index 64456a7..b93e6f5 100644
--- a/drivers/staging/Makefile
+++ b/drivers/staging/Makefile
@@ -44,4 +44,3 @@ obj-$(CONFIG_KS7010)		+= ks7010/
 obj-$(CONFIG_GREYBUS)		+= greybus/
 obj-$(CONFIG_BCM2835_VCHIQ)	+= vc04_services/
 obj-$(CONFIG_CRYPTO_DEV_CCREE)	+= ccree/
-obj-$(CONFIG_NETATOP)		+= netatop/
diff --git a/drivers/staging/netatop/Kconfig b/drivers/staging/netatop/Kconfig
deleted file mode 100644
index fa0fe16..0000000
--- a/drivers/staging/netatop/Kconfig
+++ /dev/null
@@ -1,13 +0,0 @@
-config NETATOP
-	tristate "Net statistics collector for atop"
-	default n
-	---help---
-	  netatop is used to gather statistics about the TCP and UDP packets
-	  that have been transmitted/received per process and per thread. As
-	  soon as atop discovers that this module is active, it shows the
-	  columns SNET and RNET in the generic screen for the number of
-	  transmitted and received packets per process. When the 'n' key is
-	  pressed, it shows detailed counters about the number packets
-	  transmitted/received via TCP and UDP, the average sizes of these
-	  packets, and the total bandwidth consumed for input and output per
-	  process/thread.
diff --git a/drivers/staging/netatop/Makefile b/drivers/staging/netatop/Makefile
deleted file mode 100644
index 8038ced..0000000
--- a/drivers/staging/netatop/Makefile
+++ /dev/null
@@ -1 +0,0 @@
-obj-$(CONFIG_NETATOP)	+= netatop.o
-- 
1.8.5.2.233.g932f7e4

